index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>MEdicore</title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="section--pink">
+        <div class="menu-bar">
+            <div class="menu-bar__section menu-bar__section--text-big">
+                <div class="menu-bar__link">Medical Core</div>
+            </div>
+            <div class="menu-bar__section menu-bar__section--text-muted menu-bar__links">
+                <div class="menu-bar__link">Home</div>
+                <div class="menu-bar__link">Product</div>
+                <div class="menu-bar__link">Pricing</div>
+                <div class="menu-bar__link">Contact</div>
+            </div>
+            <div class="menu-bar__section">
+                <button class="button button--white">Login</button>
+                <button class="button button--uppercase button--blue">join us <img src="./images/Vector.png"> </button>
+            </div>
+        </div>
+        <div class="welcome">
+            <div class="welcome__content">
```

Comment:
## name modifiers corresponding to what they have in CSS is not good
Naming the modifiers corresponding to their CSS representation is not recommended. Indeed it does not look very nice but there are also practical reasons against it. Later when the view of your components is changed, you will need to fix not only CSS but also the selectors. `.button-uppercase` `.button--blue` `.button-white` ... but you can follow this link [here](https://getbem.com/faq/#css-modifier-names)

Reply:
i will find new names, thanks